### PR TITLE
Easi 1934/Plan name and collaborator

### PR DESCRIPTION
**Pull request by patrickseguraoddball**
# EASI-1934

## Changes and Description

- Added components for adding a model plan, and adding/editing/removing plan collaborators
- GQL queries, mutations, and types generated
- Created custom hook for live-loading CEADR users via GQL
- Refactored component for displaying user initials within icon
- Validations written for input on forms
- Cypress testing env setup and covered model plan/collaborator flow
- Unit testing for components

## Notes

- Truss' combobox component did not fit our needs of live-loading data within a combobox 
   - Went with Reach library - accessible compliant and fit our needs
   - This 3rd party component doesn't quite match Figma.  I can do more work to simulate combobox in Figma is deemed necessary
- Edited some seeding dev data needed for cypress tests to run
- There is a test suite failing from previous PR that was using Enzyme instead of RTL - those test changes are still outstanding and will be covered in another ticket

Screen Shot 2022-04-28 at 10 34 37 AM

Screen Shot 2022-04-28 at 10 34 44 AM

Screen Shot 2022-04-28 at 10 34 52 AM

## How to test this change

To get live CEDAR data from the select, you must be connected to the VPN
Also change env to not be `TEST` or `LOCAL` or comment out this line

https://github.com/CMSgov/mint-app/blob/e1ffd9224e5dbc286a518d96033742fa3fbb9f0a/pkg/server/routes.go#L89-L91

## PR Author Review Checklist

- [ ] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [ ] Added or updated tests for backend resolvers or other functions as needed.
- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [ ] Tested user-facing changes with voice-over and the [rotor menu](https://support.apple.com/guide/voiceover/with-the-voiceover-rotor-mchlp2719/mac)

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

**Review by StevenWadeOddball — APPROVED** [review]
Looks great! I tested the changes with both CEDAR data and without it, and was able to make models and add collaborators

File: pkg/graph/schema.resolvers_helpers.go [hunk 1]
```diff
@@ -51,9 +51,9 @@ func ConvertToPlanBasics(mpi *model.PlanBasicsInput) *models.PlanBasics {
 	if mpi.ID != nil {
 		basics.ID = *mpi.ID
 	}
-	if mpi.Status != nil {
-		basics.Status = *mpi.Status
-	}
+	// if mpi.Status != nil {
+	// 	basics.Status = *mpi.Status
+	// }
```
Comment (ClayBenson94) [inline pkg/graph/schema.resolvers_helpers.go:56]: Is this intended to be commented out? Seems like with this change we'd always have a `nil` `basics.Status`

**Review by ClayBenson94 — CHANGES_REQUESTED** [review]

File: pkg/graph/schema.resolvers_helpers.go (hunk 1 above)
Comment (StevenWadeOddball) [reply pkg/graph/schema.resolvers_helpers.go:56]: yeah, I'd agrees with this, it should be commented out

**Review by StevenWadeOddball — DISMISSED** [review]

**Review by ClayBenson94 — APPROVED** [review]
This honestly looks awesome. Great job!!!

**Pull request merged**